app/medInria/medRegistrationWorkspace.cpp

```diff
@@ -66,62 +47,62 @@ void medRegistrationWorkspace::setupViewContainerStack()
     //the stack has been instantiated in constructor
     if (!this->stackedViewContainers()->count())
     {
-        d->fixedContainer = this->stackedViewContainers()->addContainerInTab(tr("Compare"));
+        d->containers.push_back(this->stackedViewContainers()->addContainerInTab(tr("Compare")));
```

Comment:
Since I feel some resistance to change in this PR :), you can leave this comment aside.